src/index.js

```diff
@@ -13,12 +13,18 @@ Gun.on('opt').event((gun, options) => {
 	const adapter = Adapter.from(level);
 
 	const { wire } = gun.__.opt;
+	var wireGet = wire.get
+	var wirePut = wire.put
 
 	// Register the driver.
 	gun.opt({
 		wire: {
-			get: wire.get || adapter.read,
-			put: wire.put || adapter.write,
+		      get: function (lex, cb, o) {
+			wireGet(lex, cb, o) || adapter.read(lex, cb)
```

Comment:
`wireGet` and `wirePut` aren't always defined, which is why [the build](https://travis-ci.org/PsychoLlama/gun-level/jobs/160968616) failed. If we make sure they're defined before calling them it should fix the errors.
We'll always want to call our own driver (`adapter.read`), but if `wire.get` is defined, then we can call both. The `wire.get` hook will likely refer to a different driver, such as websockets or json.